src/components/PlayLists/index.jsx

```diff
@@ -1,7 +1,14 @@
+import PlayListsItem from "../PlayListItem/index.jsx";
+import playlistsData from "../../../assests/json/playlists.json";
```

Comment:
Please, move the `assets` directory into the `src`.

Reply:
doneeee